redaxo/src/core/pages/setup.php

```diff
@@ -268,6 +276,16 @@
 
     $db_create_checked = rex_post('redaxo_db_create', 'boolean') ? ' checked="checked"' : '';
 
+    $httpsRedirectSel = new rex_select();
+    $httpsRedirectSel->setId('rex-form-https');
+    $httpsRedirectSel->setStyle('class="form-control selectpicker"');
+    $httpsRedirectSel->setName('use_https');
```

Comment:
Sollte man diese option im setup nur zulassen, wenn man weiß dass der server ein „funktionierendes https setup hat“, damit man sich nicht aussperrt (https erzwingt und dann in server errors läuft und kein zugriff mehr aufs redaxo)

Z.b. Könnte man diese checkbox nur zulassen wenn das setup bereits via https aufgerufen wurde